src/client/runner/runner.js

```diff
@@ -140,7 +141,7 @@ Runner.prototype._reportErrorToServer = function (err, isAssertion) {
 Runner.prototype._onTestError = function (err, isAssertion) {
     // NOTE: we should not create multiple screenshots for a step. Create a screenshot if
     // it's the first error at this step or it's an error that occurs on page initialization.
-    err.pageUrl = document.location.toString();
+    err.title = this.testRunId;
```

Comment:
Needed property is called `pageDestUrl`. https://github.com/DevExpress/testcafe/blob/master/src/client/runner/runner-base.js#L72, used in https://github.com/DevExpress/testcafe/blob/master/src/legacy/test-run-error/templates.js#L107